ONSdigital/rm-sdx-gateway
# Remove Census SFTP code

# Motivation and Context
Removing old unused SFTP code to simplify maintenance and end spurious errors in the logs.


# What has changed
All SFTP settings and methods removed, no refactoring done.




# How to test?
- `mvn clean install`
- `mvn spring-boot:run`
- deploy gateway with new image to a sandbox cluster
- run acceptance tests on sandbox cluster 
- set port forwarding from localhost with ras-rm-cli
- `curl -u $USER:$PASSWD -H "Accept: application/json" -H "Content-Type: application/json" http://localhost:8191/receipts -v -d "{\"userId\":\"ded66326-6365-4eca-afca-12bd2aca7b9b\",\"caseId\": \"069e61fe-8e87-4b50-8af9-3c4a34a643bf\"}"`
- check gateway and case service pod logs that the receipt has gone through, and check Case.Responses queue in RabbitMQ (localhost:15672)
- check that receipting works in frontstage (localhost:8082) and response-ops-ui (localhost:8085)




# Links
https://trello.com/c/MKfYULvc
